services/logging-operator/3.17.9/logging-operator.yaml

```diff
@@ -13,7 +12,7 @@ spec:
         name: kubernetes-charts.banzaicloud.com
         namespace: kommander-flux
       # When upgrading, make sure to update the fluent-bit and fluentd images in logging-operator-logging.yaml.
```

Comment:
TODO

Reply:
done in https://github.com/mesosphere/kommander-applications/pull/651/commits/4a9679559314f388a0b5b1b30b98ba78d4fc3eff